Quick/World.swift

```diff
@@ -57,12 +57,7 @@ final internal class World: NSObject {
     // MARK: Singleton Constructor
 
     private override init() {}
-    private struct Shared {
-        static let instance = World()
-    }
-    internal class func sharedWorld() -> World {
-        return Shared.instance
-    }
+    static let sharedWorld = World()
```

Comment:
If it's easy to do, it might be nicer for commit history if this change was made in a separate commit. Feel free to ignore this comment if it's too much trouble to split this commit in two.